### PR TITLE
Add support for pid ns

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
Sign this please @rhatdan

File: cmd/kpod/spec.go [hunk 1]
```diff
@@ -17,6 +18,30 @@ import (
...
+		ctr, err := config.runtime.LookupContainer(pidMode.Container())
+		if err != nil {
+			return errors.Wrapf(err, "container %q not found", pidMode.Container())
+		}
+		if _, err := ctr.State(); err != nil {
+			return err
+		}
+		return libpod.ErrNotImplemented
+		/*
+			pidNsPath := fmt.Sprintf("/proc/%d/ns/pid", state.Pid)
```
Comment (TomSweeneyRedHat) [inline cmd/kpod/spec.go]: Is this commented code needed or will it be added shortly?  A TODO comment maybe if it's going to stay?

**Comment by TomSweeneyRedHat** [comment]
Any changes/additions to the tests needed?

File: cmd/kpod/spec.go (hunk 1 above)
Comment (rhatdan) [reply cmd/kpod/spec.go]: Should add a TODO.

**Comment by rhatdan** [comment]
Yes I will add tests.

**Comment by mheon** [comment]
@rhatdan We have support baked into libpod for sharing namespaces, though it's not turned on yet - see https://github.com/projectatomic/libpod/blob/master/libpod/options.go#L300-L307

**Comment by rhatdan** [comment]
@TomSweeneyRedHat @mheon @umohnani8 @baude PTAL  Tests passing.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by rhatdan** [comment]
Removed the TODO and added handling of the state.PID call for --pid containers:UUID.

File: cmd/kpod/spec.go [hunk 2]
```diff
@@ -17,6 +17,27 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+func addPidNS(config *createConfig, g *generate.Generator) error {
```
Comment (mheon) [inline cmd/kpod/spec.go]: Can you add a TODO here to switch to the libpod support for sharing namespaces when that lands?

**Comment by mheon** [comment]
One nit, otherwise LGTM. Tests are failing though.

**Comment by umohnani8** [comment]
LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r=umohnani8

**Pull request closed without merging**